superset-frontend/spec/javascripts/explore/controlUtils_spec.tsx

```diff
@@ -209,14 +214,14 @@ describe('controlUtils', () => {
       let controlItem = findControlItem(
```

Comment:
Unfortunately no, because `findControlItem` may also return `null` or `ReactElement`. Using simple type declaration will throw an error:

<img width="964" alt="Screen Shot 2021-03-11 at 5 18 26 PM" src="https://user-images.githubusercontent.com/335541/110877160-cf02d100-828d-11eb-8a6f-dc312b9b880b.png">